### PR TITLE
UI KIT samples

### PR DESCRIPTION
**Pull request by sravan-s**
Samples for UIKIT - v1
Base apps created using create-react-app

### CODE REVIEW

File: uikit-samples/composed-react-app/src/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    marginTop: theme.spacing(8),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  main: {
+    backgroundColor: 'white',
+  },
+  form: {
+    width: '100%', // Fix IE 11 issue.
```
Comment (HoonBaek) [inline uikit-samples/composed-react-app/src/Login.jsx:27]: Is the left comment okay?

File: uikit-samples/composed-react-app/src/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        </Typography>
+        <form
+          className={classes.form}
+          onSubmit={e => {
+            e.preventDefault();
+            onSubmit({
+              userId: e.target.userId.value,
+              nickname: e.target.nickname.value,
+              theme: e.target.theme.checked ? 'dark' : 'light',
+              // useCustomQuery: e.target.useCustomQuery.checked,
```
Comment (HoonBaek) [inline uikit-samples/composed-react-app/src/Login.jsx:83]: should I  remove this?

Comment (sravan-s) [reply uikit-samples/composed-react-app/src/Login.jsx:83]: Should remove

Comment (HoonBaek) [reply uikit-samples/composed-react-app/src/Login.jsx]: I removed it

File: uikit-samples/composed-react-app/src/Login.jsx (hunk 1 above)
Comment (sravan-s) [reply uikit-samples/composed-react-app/src/Login.jsx:27]: Is from, actually good to have it here. Someone might find it useful in the future - https://material-ui.com/discover-more/showcase/

File: uikit-samples/custom-react-app/src/Chat/CustomUserList.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      },
+      {
+        userId: 'hoon101',
+        nickName: 'hoon101-display-name',
+        profileUrl: 'https://static.sendbird.com/sample/profiles/profile_12_512px.png'
+      },
+      {
+        userId: 'hoon102',
+        nickName: 'hoon102-display-name',
+        profileUrl: 'https://static.sendbird.com/sample/profiles/profile_13_512px.png'
```
[the remark this answers is not in the document]
Comment (sravan-s) [reply uikit-samples/custom-react-app/src/Chat/CustomUserList.js:29]: Yes it is 🙈

[the remark this answers is not in the document]
Comment (HoonBaek) [reply uikit-samples/custom-react-app/src/Chat/CustomUserList.js:29]: ㅋㅋㅋㅋㅋㅋ 👍

File: uikit-samples/composed-react-app/src/App.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
+}
+
+.App-link {
+  color: #61dafb;
+}
```
[the remark this answers is not in the document]
Comment (sravan-s) [reply uikit-samples/composed-react-app/src/App.css:29]: Those were from create-react-app we can delete those

**Pull request merged**